docs/guides/docs/running-a-node/running-a-beta-4-node.mdx

```diff
@@ -0,0 +1,304 @@
+    "gm": 1,
+    "gt": 1,
+    "gtf": 1,
+    "ji": 1,
+    "jmp": 1,
+    "jne": 1,
+    "jnei": 1,
+    "jnzi": 1,
+    "jmpf": 1,
+    "jmpb": 1,
+    "jnzf": 1,
+    "jnzb": 1,
+    "jnef": 1,
+    "jneb": 1,
+    "k256": 11,
+    "lb": 1,
+    "log": 9,
+    "lt": 1,
+    "lw": 1,
+    "mcpi": 33,
+    "mint": 135,
+    "mlog": 1,
+    "mod": 1,
+    "modi": 1,
+    "move": 1,
+    "movi": 1,
+    "mroo": 2,
+    "mul": 1,
+    "muli": 1,
+    "mldv": 1,
+    "noop": 1,
+    "not": 1,
+    "or": 1,
+    "ori": 1,
+    "ret_contract": 13,
+    "rvrt_contract": 13,
+    "s256": 2,
+    "sb": 1,
+    "scwq": 13,
+    "sll": 1,
+    "slli": 1,
+    "srl": 1,
+    "srli": 1,
+    "srw": 12,
+    "sub": 1,
+    "subi": 1,
+    "sw": 1,
+    "sww": 43,
+    "swwq": 44,
+    "time": 1,
+    "tr": 105,
+    "tro": 60,
+    "wdcm": 1,
+    "wqcm": 1,
+    "wdop": 1,
+    "wqop": 1,
+    "wdml": 1,
+    "wqml": 1,
+    "wddv": 1,
+    "wqdv": 2,
+    "wdmd": 3,
+    "wqmd": 4,
+    "wdam": 2,
+    "wqam": 3,
+    "wdmm": 3,
+    "wqmm": 3,
+    "xor": 1,
+    "xori": 1,
+    "call": {
+      "base": 144,
+      "dep_per_unit": 214
+    },
+    "ccp": {
+      "base": 15,
+      "dep_per_unit": 103
+    },
+    "csiz": {
+      "base": 17,
+      "dep_per_unit": 790
+    },
+    "ldc": {
+      "base": 15,
+      "dep_per_unit": 272
+    },
+    "logd": {
+      "base": 26,
+      "dep_per_unit": 64
+    },
+    "mcl": {
+      "base": 1,
+      "dep_per_unit": 3333
+    },
+    "mcli": {
+      "base": 1,
+      "dep_per_unit": 3333
+    },
+    "mcp": {
+      "base": 1,
+      "dep_per_unit": 2000
+    },
+    "meq": {
+      "base": 1,
+      "dep_per_unit": 2500
+    },
+    "retd_contract": {
+      "base": 29,
+      "dep_per_unit": 62
+    },
+    "smo": {
+      "base": 209,
+      "dep_per_unit": 55
+    },
+    "srwq": {
+      "base": 47,
+      "dep_per_unit": 5
+    }
+  },
+  "consensus": {
+    "PoA": {
+      "signing_key": "f65d6448a273b531ee942c133bb91a6f904c7d7f3104cdaf6b9f7f50d3518871"
+    }
+  }
+}
+```
+
+## Running a Local Node
+
+Finally to put everything together to start the node, run the following command:
+
+```sh
+$ fuel-core run \
+--service-name {ANY_SERVICE_NAME} \
+--keypair {P2P_SECRET} \
+--relayer {RPC_ENDPOINT} \
+--ip 127.0.0.1 --port 4000 --peering_port 30333 \
+--db-path  ~/.fuel_beta4 \
+--chain ./chainConfig.json \
+--utxo-validation --poa-instant false --network beta-4 --enable-p2p \
+--min-gas-price 1 --max_block_size 18874368  --max_transmit_size 18874368 \
+--bootstrap_nodes /dns4/p2p-beta-4.fuel.network/tcp/30333/p2p/16Uiu2HAm3xjsqASZ68KpaJPkPCMUiMgquhjyDHtxcVxVdFkMgRFf,/dns4/p2p-beta-4.fuel.network/tcp/30334/p2p/16Uiu2HAmJyoJ2HrtPRdBALMT8fs5Q25xVj57gZj5s6G6dzbHypoS \
+--sync_max_get_header 100 --sync_max_get_txns 100 \
+--relayer-v2-listening-contracts 0x03f2901Db5723639978deBed3aBA66d4EA03aF73 \
+--relayer-da-finalization 4 \
+--relayer-da-deploy-height 4111672 \
+--relayer-log-page-size 10000
```

Comment:
Now that the relayer is explicitly enabled, you will need to add `--enable-relayer` to your list of arguments.

Reply:
This change is not part of the Beta-4, so you can't use it=) But in the later releases - yes, we need to update that

Reply:
Added a comment for this. Thanks for the note @bvrooman